sdk/src/index.ts

```diff
@@ -1,43 +0,0 @@
-import { Keyring } from '@polkadot/api'
```

Comment:
We'll ultimately remove this file, but for now, it's more convenient to leave it